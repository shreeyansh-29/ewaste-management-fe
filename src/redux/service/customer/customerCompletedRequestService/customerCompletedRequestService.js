import {CUSTOMER_REQUEST_COMPLETED} from "../../../../container/constant/constant";
import api from "../../../../core/utilities/httpProvider";

export const customerCompletedRequestService = () => {
  return api.get(CUSTOMER_REQUEST_COMPLETED);
};
