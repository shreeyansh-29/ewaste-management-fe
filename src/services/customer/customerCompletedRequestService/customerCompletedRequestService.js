import {CUSTOMER_REQUEST_COMPLETED} from "../../../container/constant/constants";
import api from "../../../core/utilities/httpProvider";

export const customerCompletedRequestService = () => {
  return api.get(CUSTOMER_REQUEST_COMPLETED);
};
