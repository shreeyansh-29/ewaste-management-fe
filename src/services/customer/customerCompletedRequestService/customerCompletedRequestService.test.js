/* eslint-disable no-undef */

import {customerCompletedRequestService} from "./customerCompletedRequestService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing customerCompletedRequestService", () => {
  const Service = customerCompletedRequestService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
