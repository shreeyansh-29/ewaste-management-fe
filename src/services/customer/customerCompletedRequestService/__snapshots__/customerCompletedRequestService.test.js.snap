// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing customerCompletedRequestService 1`] = `<Component />`;
